src/sentry/plugins/providers/integration_repository.py

```diff
@@ -88,15 +78,33 @@ def handle_api_error(self, error):
         context = {
             'error_type': 'unknown',
         }
+
         if isinstance(error, IntegrationError):
-            # TODO(dcramer): we should have a proper validation error
+            if '503' in error.message:
+                context.update({
+                    'error_type': 'service unavailable',
+                    'errors': {
+                        '__all__': error.message
+                    },
+                })
+                status = 503
+            else:
+                # TODO(dcramer): we should have a proper validation error
+                context.update({
+                    'error_type': 'validation',
```

Comment:
I can't completely tell where `error_type` is coming from or if I would need it. I saw various examples in code, but cannot figure out what the pattern is. @markstory any thoughts?

Reply:
I understand but is there a place where this is used and/or what are the options for `error_type`

Reply:
Or is it just whatever goes...? I'm assuming at this stage it's a UI thing.

Reply:
Yeah these go out in the API response afaik.